.travis.yml

```diff
@@ -1,16 +1,28 @@
 language:
   ruby
 
+env:
+  - CPLUS_INCLUDE_PATH=/usr/include/atlas C_INCLUDE_PATH=/usr/include/atlas
+
 rvm:
-  - 1.9.3
-  - 2.0
-  - 2.1
-  - 2.2
+  - '1.9.3'
+  - '2.0'
+  - '2.1'
+  - '2.2'
+
+matrix:
+  fast_finish:
+    true
+
+script: "bundle exec rake test"
+
+install:
+  - gem install bundler
+  - ./.build.sh
+  - bundle install
```

Comment:
The build script only clones and installs nmatrix and gsl-nmatrix. The rest of the dependencies need to be done with bundle install.

Also a couple of gems in the Gemfile need to be checking in before usage, which makes bundle install necessary.